specifyweb/frontend/js_src/lib/components/Statistics/StatsSpec.tsx

```diff
@@ -336,7 +329,7 @@ export const statsSpec: StatsSpec = {
           },
         },
       },
-
+      /*
```

Comment:
Can you include a FIXME comment here? 
As we have done for similar feature cuts in the past.

Reply:
There should be one for removing taxa stats right? They both will be fixed the same time, so thought one won't be necessary here.  I'll still add one.

Reply:
Fair enough. Just being thorough :+1: